.editorconfig

```diff
@@ -3,10 +3,12 @@ root = true
 [*]
 charset = utf-8
 end_of_line = lf
-indent_size = 2
+indent_size = 4
 indent_style = space
 insert_final_newline = true
 trim_trailing_whitespace = true
+dotnet_style_operator_placement_when_wrapping = beginning_of_line
+tab_width = 4
```

Comment:
What was the motivation for these changes? Can they be moved to an extension-specific section?